src/types.ts

```diff
@@ -1,12 +1,14 @@
 import * as _ from 'lodash'
 import { ScenarioApi } from "./api"
-import * as t from "io-ts"
+// import * as t from "io-ts"
 import { reporter } from 'io-ts-reporters'
-import { ThrowReporter } from "io-ts/lib/ThrowReporter"
-import { ChildProcess } from 'child_process';
+// import { ThrowReporter } from "io-ts/lib/ThrowReporter"
+// import { ChildProcess } from 'child_process';
```

Comment:
io-ts generates helpful and readable runtime validators of data shape, so you can check that a deeply nested data structure matches the typescript type, so the app dev can get more sensible feedback, instead of getting a much less helpful error from msgpack when you try to decode something invalid later. I'd consider putting it back in at some point.

Reply:
Yeah, we would put it back in, during the part where we re-enable conductor configuration overrides, and sugar functions for generating `InstallAgentHapps`